### ISSUE TITLE
Bazarr freezes on saving language profile

### ISSUE THREAD

**Issue by ianepreston** [morpheus65535/bazarr#1812]
**Describe the bug**
When I go to create a language profile and hit save, the save button spins indefinitely.

**To Reproduce**
* Go to settings
* Go to languages
* Click "Add new profile"
* Populate the field
* Hit save so the popup closes
* Hit save again to save the profile

**Expected behavior**
Should actually save, just spins on saving indefinitely

**Software (please complete the following information):**
 - Bazarr: 1.0.3
 - Radarr version 4.1.0.6175
 - Sonarr version 3.0.7.1477
 - OS: docker container from linuxserver, Linux-5.15.26-1-lts-x86_64-with

**Additional context**
Traceback in the log:

```python
Traceback (most recent call last):

  File "/app/bazarr/bin/bazarr/../libs/flask/app.py", line 1516, in full_dispatch_request

    rv = self.dispatch_request()

  File "/app/bazarr/bin/bazarr/../libs/flask/app.py", line 1502, in dispatch_request

    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)

  File "/app/bazarr/bin/bazarr/../libs/flask_restful/__init__.py", line 467, in wrapper

    resp = resource(*args, **kwargs)

  File "/app/bazarr/bin/bazarr/../libs/flask/views.py", line 84, in view

    return current_app.ensure_sync(self.dispatch_request)(*args, **kwargs)

  File "/app/bazarr/bin/bazarr/../libs/flask_restful/__init__.py", line 582, in dispatch_request

    resp = meth(*args, **kwargs)

  File "/app/bazarr/bin/bazarr/api/utils.py", line 30, in wrapper

    return actual_method(*args, **kwargs)

  File "/app/bazarr/bin/bazarr/api/system/settings.py", line 73, in post

    TableLanguagesProfiles.mustContain: item[\'mustContain\'],

KeyError: \'mustContain\
```

**Comment by morpheus65535** [comment]
Should have been fixed in v1.0.4-beta.21. Can you test and confirm please?

**Comment by ianepreston** [comment]
Confirmed fixed. Thank you!
